src/JHipsterNetSampleApplication/Models/Interfaces/IJoinEntity.cs

```diff
@@ -0,0 +1,6 @@
+
+namespace JHipsterNetSampleApplication.Models.Interfaces {
+    public interface IJoinEntity<TEntity> {
+        TEntity Navigation { get; set; }
```

Comment:
`Navigation` vs `Join`? What makes more sense?

Reply:
I changed to Join